Source/controls/plrctrls.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 // Controller actions implementation
 
+#include "player.h"

```

Comment:
Our includes should come after:

1. Standard library
2. Other libraries
2. Ours

Reply:
I know. It turns out clang-format places them in this order them if you don't add a blank line in between.